### PR TITLE
Add ci builds

### LINKED ISSUE #7: Add CI build
Add a CI build that compiles the source and execute the tests.

### PR DESCRIPTION
**Pull request by discontinuedtv**

### CODE REVIEW

File: BotBrownCore/Workers/TextToSpeech/TextToSpeechProcessor.cs [hunk 1]
```diff
@@ -1,150 +1,150 @@
...
+    using NAudio.Wave;
+
+    public class TextToSpeechProcessor : ITextToSpeechProcessor
+    {
+        private const int BitResolution = 16;
+
+        private readonly IConfigurationManager configurationManager;
+        private readonly IDictionary<string, string> availableLanguages = new Dictionary<string, string>();
+        private readonly AudioConfiguration audioConfiguration;
+        private bool isInitialized;
```
Comment (i-am-steve-oh) [inline BotBrownCore/Workers/TextToSpeech/TextToSpeechProcessor.cs:23]: isInitialized isn't used actually. Didn't we dropped the init process?

**Review by i-am-steve-oh — APPROVED** [review]
I would merge this request an remove the unused parts in another pull request actually.

File: BotBrownCore/Workers/TextToSpeech/TextToSpeechProcessor.cs (hunk 1 above)
Comment (discontinuedtv) [reply BotBrownCore/Workers/TextToSpeech/TextToSpeechProcessor.cs:23]: Yes. I think we did. I'll fix it with the issue regarding the other warnings.

**Comment by discontinuedtv** [comment]
Sounds good. Thank you.

**Pull request merged**